<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connecet_mainwindow</class>
 <widget class="QMainWindow" name="connecet_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HamWAN Node Tool - Connect</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="connect_pushbutton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ip_address_label">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="user_label">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ip_address_lineedit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="user_lineedit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="password_lineedit">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>HamWAN Node Tool is used to aid in the configuration of end-user modems. It also contains tools to help you align your antenna to the cell site.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
